.github/workflows/check-links.yml

```
@@ -13,7 +13,9 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v4
-      - uses: gaurav-nelson/github-action-markdown-link-check@v1
+      - name: Run markdown link check
+        uses: gaurav-nelson/github-action-markdown-link-check@v1
         with:
           use-quiet-mode: yes
           use-verbose-mode: yes
+          config-file: .mlc_config.json
```

Looks like it should be possible to use the [custom variables](https://github.com/marketplace/actions/markdown-link-check#custom-variables) `folder-path` and `file-path` together to only include what we want and not need to ignore individual urls.

Does this work?
```suggestion
          config-file: .mlc_config.json
          file-path: './README.md, ./CONTRIBUTING.md'
          folder-path: 'docs'
```